tests/test_package_client_cpp.py

```diff
@@ -230,11 +231,6 @@ def test_install_configure_start(
 
         self.check_mender_client_version(setup_tester_ssh_connection, mender_version)
 
-        # Default setup expects ServerURL hosted.mender.io
-        # TODO - No mender-setup yet
-        # result = setup_tester_ssh_connection.sudo("cat /etc/mender/mender.conf")
-        # assert '"ServerURL": "https://hosted.mender.io"' in result.stdout
```

Comment:
Why don't we want to test this?

Reply:
It is tested for `mender-setup` package. See [here](https://github.com/mendersoftware/mender-dist-packages/blob/3ad8a339919bfbf57d1e420d5db8e8f6fb28fd5f/tests/test_package_setup.py#L60).

Reply:
Right, of course!